### PR TITLE
Add minor ace fixes

### PR DESCRIPTION
**Pull request by anthonyprinaldi**
Add:
- Ability to skip segmentation
- Voxelization and warping to single subject ace workflow
- Default value of allen label atlas for ace registration

### CODE REVIEW

File: miracl/reg/miracl_reg_clar-allen.sh [hunk 1]
```diff
@@ -463,7 +463,7 @@ function resampleclar()
 	ifdsntexistrun ${resclar} "Resampling CLARITY input" \
 	ResampleImage 3 ${inclar} ${resclar} ${vox}x${vox}x${vox} ${ifspacing} ${interp}
 
-	c3d ${resclar} -type ushort -o ${resclar}
+	c3d ${resclar} -type ushort -o /code/clar.nii.gz
```
Comment (jono3030) [inline miracl/reg/miracl_reg_clar-allen.sh:466]: What is this doing again? Is it saving sth into `/code`? Also, do we have an env var for `/code`? If not, should we create one to use here?

File: miracl/flow/miracl_workflow_ace_interface.py [hunk 2]
```diff
@@ -801,13 +852,71 @@ def _clear_reg_folders(reg_folder: Path):
...
         reg_folder.mkdir(parents=True, exist_ok=True)
 
-        # TODO: do we also want to remove this dir?
         clar_allen_reg = reg_folder.parent / "clar_allen_reg"
         if clar_allen_reg.is_dir():
             shutil.rmtree(clar_allen_reg)
         clar_allen_reg.mkdir(parents=True, exist_ok=True)
 
 
+class SegmentationChecker:
```
Comment (jono3030) [inline miracl/flow/miracl_workflow_ace_interface.py:861]: This is great!

File: miracl/reg/miracl_reg_clar-allen.sh (hunk 1 above)
Comment (anthonyprinaldi) [reply miracl/reg/miracl_reg_clar-allen.sh:466]: this is just a temporary change. Registration is giving us problems so needed to check intermediate outputs. Should have done it on a branch not already in a PR...

Comment (jono3030) [reply miracl/reg/miracl_reg_clar-allen.sh:466]: Ah of course, I could have guessed that as well that it's debugging...do you want to hop on Zoom re: the reg issue?

**Comment by anthonyprinaldi** [comment]
Close PR and move to dev branch.

**Comment by anthonyprinaldi** [comment]
Close PR and move to dev branch.

**Pull request closed without merging**